client/app.html

```diff
@@ -15,6 +15,7 @@
     <script src="../assets/questions/parens.js"></script>
     <script src="../assets/questions/reverseWords.js"></script>
     <script src="../assets/questions/rle.js"></script>
+   
```

Comment:
Why did you add an empty space, here? And have you imported your new question anywhere?